app/models/katello/repository.rb

```diff
@@ -302,7 +302,7 @@ def on_demand?
     end
 
     def immediate?
-      root.download_policy == ::Katello::RootRepository::DOWNLOAD_ON_IMMEDIATE
+      root.download_policy == ::Katello::RootRepository::DOWNLOAD_IMMEDIATE
```

Comment:
Why change this?

Reply:
Oh that was a buggy code. Some one changed the constant name from  `DOWNLOAD_ON_IMMEDIATE`  to `DOWNLOAD_IMMEDIATE` but forgot to change it in the model here

Reply:
Actually I don't see any code calling "immediate?" so maybe it's not urgent

Reply:
https://github.com/Katello/katello/pull/9925/files#diff-6dd39994c7a6695008f92af2b9727cabe5ce0f3aa74e4b00776bab6f7f228b2cR42
look at `validate_repositories_immediate` in `export_repository` action

Reply:
ACK. Last thing - is this change compatible with 6.10 ie when did the constant reference get changed/broken ?